froide/problem/utils.py

```diff
@@ -23,6 +26,14 @@ def inform_managers(report):
     )
 
 
+def can_read(user, foirequest):
+    request_factory = RequestFactory()
```

Comment:
Request is needed because of possible session tokens and OAuth stuff that doesn't live on the user. But maybe we need a helper that makes creating a fake request for a user more convenient.